service/arm_templates/nodes_setup.json

```diff
@@ -0,0 +1,141 @@
+{
```

Comment:
I think this file should be called `node_setup.json` asa it creates  a single node.